### PR TITLE
Fix formatting issues in analytics documentation

### LINKED ISSUE #533: Fix formatting issues in analytics documentation

### PR DESCRIPTION
**Pull request by sgayangi**
Fixes #533

### CODE REVIEW

**Review by CrowleyRajapakse — APPROVED** [review]

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 1]
```diff
@@ -2,46 +2,69 @@
 
 ## Step 1 - Setup APK
 
-1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file, which you will then use to tailor the analytics configurations to your specific needs. Open `values.yaml` file.
+1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file.
+2. Open the `values.yaml` file, and add the above configuration to the gatewayRuntime section under dp.
```
Comment (Tharanidk) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:6]: `gatewayRuntime` and `dp`
better to highlight these two words

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 2]
```diff
@@ -2,46 +2,69 @@
...
+     publishers:
+       - enabled: true
+         type: "default"
+         secretName: <choreo-secret-name>
+       - enabled: true
+         type: "elk"
+       - enabled: true
+         type: "moesif"
+         secretName: <moesif-secret-name>
+```
```
Comment (Tharanidk) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:66]: remove extra blank line

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md (hunk 1 above)
Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:6]: Fixed in https://github.com/wso2/docs-apk/pull/583

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md (hunk 2 above)
Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:66]: Fixed in PR #583

**Pull request merged**